### PR TITLE
[Project] Take Job as param by invoke()

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's take Job as param by invoke().

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <<yons.kim@samsung.com>>

### CODE REVIEW

**Comment by YongseopKim** [comment]
I don't yet test this with `onecc`. This is necessary for the test.

File: src/Project/ToolRunner.ts [hunk 1]
```diff
@@ -83,12 +84,10 @@ export class ToolRunner {
     return oneccRealPath;
   }
 
-  public getRunner(name: string, onecc: string, tool: string, toolargs: ToolArgs, path: string) {
+  public getRunner(job: Job, path: string) {
```
Comment (hyunsik-yoon) [inline src/Project/ToolRunner.ts:87]: In `JobRunner.ts`, it is reasonable to change previous code to getRunner(job, path) but
in `ToolRunner.ts`, I am not sure this needs to take `job` as parameter. 
IMHO, I guess `ToolRunner` handles level lower underneath `JobRunner`, and `ToolRunner` is used by `JobRunner` or some other classes which want to run some tool with string params.

Comment (YongseopKim) [reply src/Project/ToolRunner.ts:87]: > I guess ToolRunner handles level lower underneath JobRunner, and ToolRunner is used by JobRunner or some other classes which want to run some tool with string params.

That's very reasonable. I'll fix it. And FYI the reason why I changed is to ease make it call the callback function.

Comment (YongseopKim) [reply src/Project/ToolRunner.ts:87]: Now I see that fixing `toolrunner` is not necessary. Thanks!

File: src/Project/ToolRunner.ts [hunk 2]
```diff
@@ -83,12 +83,10 @@ export class ToolRunner {
     return oneccRealPath;
   }
 
-  public getRunner(name: string, onecc: string, tool: string, toolargs: ToolArgs, path: string) {
+  public getRunner(name: string, tool: string, toolargs: ToolArgs, path: string) {
```
Comment (YongseopKim) [inline src/Project/ToolRunner.ts:86]: This change is necessary because now this `ToolRunner` is very dependent on `onecc` as a driver.

**Comment by YongseopKim** [comment]
I confirm the tests are passed with onecc.

**Review by hyunsik-yoon — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**